content/applications/sales/point_of_sale/shop/customer_display.rst

```diff
@@ -2,72 +2,102 @@
+during the checkout process. This screen shows the :ref:`items in the cart <pos/sell>`, the subtotal
+as items are added, and details throughout the payment process. It also displays the total amount,
+the selected :doc:`payment method <../payment_methods>`, and any change to be returned.
 
 .. image:: customer_display/display.png
    :alt: customer screen
+   :scale: 50 %
+
+.. note::
+   Both the customer display and POS display should be at least 6 inches. However, a larger screen
+   is recommended for improved end-user readability.
 
 Configuration
 =============
 
-Depending on your POS setup, the feature can be displayed :ref:`locally on a secondary screen
-<customer_display/local>`, :ref:`remotely on another device <customer_display/remote>`, or
-:ref:`another monitor connected to an IoT Box <customer_display/iot>`.
+Depending on the POS setup, feature can be displayed :ref:`directly on a secondary screen
+<customer_display/local>` connected via USB-C or HDMI, :ref:`on another device remotely
+<customer_display/remote>`, or :ref:`on a screen connected through an IoT system
+<customer_display/iot>`.
+
+To activate the feature, follow these steps:
+
+#. Navigate to the :ref:`POS settings <configuration/settings>`.
+#. Scroll down to the :guilabel:`Connected Devices` section.
+#. Open the dropdown menu under the :guilabel:`Customer Display` section and select one of the
+   following options:
+
+   - :guilabel:`None`: To disable the customer display.
+   - :guilabel:`The same device`: To use the secondary screen connected with an HDMI or USB-C cable.
+   - :guilabel:`Another device`: To connect to a remote device connected to the database.
+   - :guilabel:`IOT-connected screen` for displays connected via an :doc:`IoT system
+     <../../../general/iot>`.
 
-To activate the feature, go to the POS settings, scroll down to the :guilabel:`Connected Devices`
-section, and tick the :guilabel:`Customer Display` checkbox.
+For displays connected using an :doc:`IoT system <../../../general/iot>`:
 
-.. image:: customer_display/feature-setting.png
-   :alt: customer display setting checkbox
+#. From the same section, tick the :guilabel:`IoT Box` checkbox to activate the IoT system in POS.
+#. Select the IoT-connected screen from the dropdown menu in the :guilabel:`Customer Display` field.
+
+Reaching the customer display
+=============================
 
 .. _customer_display/local:
 
 Local
 -----
 
-Connect a second screen to your POS using an HDMI or USB-C cable, then,
+To open the customer display on a second screen connected to a POS system using an HDMI or USB-C
+cable, follow these steps:
 
 #. :ref:`Open a POS session <pos/session-start>`.
 #. Click the :icon:`fa-bars` icon (:guilabel:`hamburger menu`).
-#. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`) to open a new window to drag and
-   drop onto the second screen.
+#. Click the :icon:`fa-desktop` icon (:guilabel:`customer screen`), which opens a new window to drag
```

Comment:
```suggestion
#. Click the :icon:`fa-desktop` (:guilabel:`Customer Display`) icon, which opens a new window to drag
```
I recommend using the tooltip as it is (when there is one obviously :sweat_smile:)